doc/design/Benchmark/kubemark/kubemark-benchmarking.md

```diff
@@ -0,0 +1,44 @@
+## BACKGROUND
+
+Performance comparison of KubeBatch scheduler with default scheduler. Kubemark will be used as a tool for benchmarking.
+Kubemark is a performance testing tool which we use to run simulated kubernetes clusters. The primary use case is scalability 
+testing, as creating simulated clusters is faster and requires less resources than creating real ones.
+
+## OBJECTIVE
+
+- Comparing the scheduler performance of kube-batch with the scheduler performance of the default scheduler
```

Comment:
Our objective is to evaulate performance of kube-batch; it's not necessary to compare with default scheduler.